examples/python/template/main.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+"""
+Example template.
+
+Run:
+```sh
+pip install -r examples/python/template/requirements.txt
+./examples/python/template/main.py
```

Comment:
In a context of examples and when the use of venv is mandated, I'd say it's more standard to call `python` explicitly:

```
python examples/python/template/main.py
```

I'd also remove the `#!` line altogether. It's weird outside of "script" context.

Reply:
I think keeping the shebang is fine, since it doesn't hurt anything, and some people may want to run the example directly as if it was an executable